rules.mk

```diff
@@ -0,0 +1,7 @@
+# This file should be included in user layout's rules.mk file
+LIB_SRC += sequence_transform/sequence_transform.c
+LIB_SRC += sequence_transform/utils.c
+LIB_SRC += sequence_transform/trie.c
+LIB_SRC += sequence_transform/keybuffer.c
+LIB_SRC += sequence_transform/key_stack.c
+LIB_SRC += sequence_transform/cursor.c
```

Comment:
This is great. It will be so nice not having to add and remove these when switching back and forth between branches.

I verified that this works once I figured out that it needed the full path in my configuration